### PR TITLE
feat(suite): label translations

### LINKED ISSUE #7624: Label community translations
Add new community translations
  - Italian (GiorgioHerbie, blablableble)
  - Hungarian (Mario Goomba)
    
- Add community label to these languages
  -  Italian
  - Hungarian
  - Japanese
  - Russian
  - Spanish
  - Chinese
  
  _Officially supported languages are currently Czech and English_

Final design: [Figma](https://www.figma.com/file/T0DBX3enk49MAbGx7JjPpd/Suite-Desktop-PO-Workspace?type=design&node-id=41%3A230&mode=design&t=dzuknoYIDBLh5Kt5-1)

### PR DESCRIPTION
**Pull request by AdamSchinzel**
Translations are divided into groups.

## Description

Translations are divided into official and community group (grouping checked with @Hermez-cz). Removed group divider.

Added translation keys:
- `TR_LANGUAGE_DESCRIPTION`
- `TR_LANGUAGE_CREDITS`
- `TR_OFFICIAL_LANGUAGES`
- `TR_COMMUNITY_LANGUAGES`

## Related Issue

Resolve #7624 

## Screenshots:

Screenshot 2023-08-11 at 12 45 41 PM

### CODE REVIEW

File: packages/suite/src/views/settings/general/Language.tsx
Comment (komret) [inline packages/suite/src/views/settings/general/Language.tsx:1]: According to the [Figma design](https://www.figma.com/file/T0DBX3enk49MAbGx7JjPpd/Suite-Desktop-PO-Workspace?type=design&node-id=39-159&mode=design&t=p8x8OWzW8D3ZMjQ7-0), there should be the system language in parenthesis.
Screenshot 2023-08-14 at 12 39 41

**Comment by komret** [comment]
@sime @Hermez-cz I find the copy somewhat confusing:
- "The Trezor community translated Czech..." - and yet, we call it "official", not "community". Also, we only mention Czech while there are more translations by the community. Also, most users don't speak Czech, but we show this to all of them.
- "Ensure language consistency by referring to our official languages for verification." - I find this sentence hard to understand.
- "See credits": The linked page does not show credits.

File: packages/components/src/components/form/Select/index.tsx [hunk 1]
```diff
@@ -153,7 +153,6 @@ const selectStyle = (
         ...base,
         padding: 0,
         '& + &': {
-            borderTop: `1px solid ${theme.BG_WHITE_ALT_HOVER}`,
```
Comment (komret) [inline packages/components/src/components/form/Select/index.tsx:156]: Was this removed from the design on purpose? I think it is clearer with the separator.

**Review by komret — APPROVED** [review]
Please, squash the commits into one.

I have some doubts regarding the design/copy, but code is fine.

File: packages/suite/src/views/settings/general/Language.tsx
Comment (AdamSchinzel) [reply packages/suite/src/views/settings/general/Language.tsx:1]: Discussed with @Hannsek and if we will do it like this we should do it also for theme (`system(light/dark)`), right now not going to change this.

**Comment by Hermez-cz** [comment]
@komret thank you for your feedbacks.

- The "Community language" text in the design is only an example (example in English about Czech for everyone here to understand the meaning). It should be shown only when a community language is selected and it should be shown in that very language and speaking about he selected language (= if user selects an official language, no such text should be shown).
- "Ensure language consistency by referring to our official languages for verification." I'll check if we might simplify while keeping it short.
- It's hard to lead directly to "Credits" on Crowdin, so we have included a link just to the project, where the users can find it (especially when logged in). We might direct the link to https://crowdin.com/project/trezor-suite/activity-stream but it's a bit fuzzy.

**Comment by Hermez-cz** [comment]
Let's move the Czech language to community languages please. That will leave us only with English in the official ones, but that will change soon. Czech language is translated very well, but by the community incl. our internal colleagues. We will move it to the official once it will be professionally proofread.

**Comment by Hermez-cz** [comment]
**Final & agreed solution:**

- The description text will only be shown when one of the community languages will be selected, that means it will be shown in that very language
- Final copy in English (only as source for translations, it will never be shown as english is an official language): A big thanks to the Trezor community for helping out with this translation. You can always refer to one of the official languages if needed.
- The input and the button will have the same width to stay aligned (✅ solved in #9233 )

**Comment by hrudolf** [comment]
I'm happy that this issue got a solution and community languages got closer to deployment. I've posted a question to TR_LANGUAGE_DESCRIPTION on Crowdin if the Nationality in the message should be translated to the selected country's nationality (eg. it's Cesky for Czech, magyar for Hungarian). I think that's the intention, right? Could someone from Trezor please add an answer there so that other translators may translate it accordingly? Thanks!

**Pull request merged**